app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -85,11 +92,25 @@ fun TripItem(trip: Trip) {
   Card(
       modifier = Modifier.fillMaxSize().testTag("cardItem"),
       content = {
-        Text(
-            text = trip.name,
-            modifier = Modifier.padding(16.dp),
-            fontSize = 20.sp,
-            textAlign = TextAlign.Center)
-      },
-  )
+        Row {
+          if (trip.imageUri.isNotEmpty()) {
+            Image(
+                painter = rememberAsyncImagePainter(model = trip.imageUri),
+                contentDescription = "Selected image",
+                contentScale = ContentScale.Crop,
+                modifier = Modifier.width(120.dp).height(217.dp))
```

Comment:
You don't necessarily have to address this comment now but,
These fixed values for width and height made me think about what would happen on larger/smaller screens, maybe the image would appear too big/too small. 
Not sure if this is a singular case, we could have used fixed values in other files as well, so we should test the whole app on other device resolutions as well. I'll add this as a task on the Scrum Board, it's probably not doable until M1 but we should consider it for future Sprints.

Reply:
+1

Reply:
For this PR let's prioritize merging the basic functionalities for the 1st milestone. We can address this in a future PR.